### ISSUE TITLE
Problem extracting gz files

### ISSUE THREAD

**Issue by Nic-Nic** [saezlab/pypath#45]
I was trying some of the PyPath methods to load some data (eg: pa.sequences, pa.load_ptms,...) and there seems to be a problem when processing the data requested, seems to be on the extraction process. Here the last part of the traceback:

```python
/home/nico/.local/lib/python2.7/site-packages/pypath/seq.pyc in get_isoforms(organism)
     82     result = {}
     83     url = urls.urls['unip_iso']['url']
---> 84     c = curl.Curl(url, silent=False)
     85     data = c.result
     86     data = read_fasta(data)

/home/nico/.local/lib/python2.7/site-packages/pypath/curl.pyc in __init__(self, url, silent, get, post, req_headers, cache, debug, outf, compr, encoding, files_needed, timeout, init_url, init_fun, init_use_cache, follow, large, override_post, init_headers, return_headers, compressed, binary_data, write_cache, force_quote, sftp_user, sftp_passwd, sftp_passwd_file, sftp_port, sftp_host, sftp_ask, setup, call, process, retries, cache_dir)
    766             sys.stdout.flush()
    767         if process and not self.download_failed and not DRYRUN:
--> 768             self.process_file()
    769 
    770         if DRYRUN:

/home/nico/.local/lib/python2.7/site-packages/pypath/curl.pyc in process_file(self)
   1208         self.copy_file()
   1209         self.open_file()
-> 1210         self.extract_file()
   1211         self.decode_result()
   1212         self.report_ready()

/home/nico/.local/lib/python2.7/site-packages/pypath/curl.pyc in extract_file(self)
   1230         if not self.silent:
   1231             self.print_status('Extracting %s data' % self.type)
-> 1232         self.extract()
   1233 
   1234     def decode_result(self):

/home/nico/.local/lib/python2.7/site-packages/pypath/curl.pyc in extract(self)
    560 
    561     def extract(self):
--> 562         getattr(self, 'open_%s' % self.type)()
    563 
    564     def open_tgz(self):

/home/nico/.local/lib/python2.7/site-packages/pypath/curl.pyc in open_gz(self)
    583 
    584     def open_gz(self):
--> 585         self.fileobj.seek(-4, 2)
    586         self.size = struct.unpack('I', self.fileobj.read(4))[0]
    587         self.fileobj.seek(0)

IOError: [Errno 22] Invalid argument
```

**Comment by deeenes** [comment]
I can't reproduce this. Are you sure the gz file is intact? What is you call in context `pypath.curl.cache_off()`?

**Comment by deeenes** [comment]
Thanks Nico, I think the 2 above just have been fixed in `0.7.101`
